README.md

```diff
@@ -221,6 +246,13 @@ won't help you, and you'll need to write a lower-level CMake
 file. This README can't cover all the possible situations here, but
 the `nowide` and `catch` CMake files are both solid examples.
 
+## How To Release
+
+1. Update [CHANGELOG.md](CHANGELOG.md) with release notes based on
+``git log `git describe --abbrev=0 --tags`..HEAD``
+1. Update the version in the project declaration of [CMakeLists.txt](CMakeLists.txt)
```

Comment:
Sorry it looks like my comment got cut off. I meant to say "... and how you can mark a release as backward incompatible here https://github.com/puppetlabs/leatherman/blob/master/LeathermanConfigVersion.cmake.in"